locale/ca_ES.ts

```diff
@@ -627,7 +627,7 @@ entre l&apos;ortografia clàssica i l&apos;escolar en ciríl·lic)</translation>
     <message>
         <location filename="../dictheadwords.ui" line="53"/>
         <source>Exports headwords to file</source>
-        <translation>Exporta les paraules principals a un fitxer</translation>
+        <translation>Exporta les entrades a un fitxer</translation>
```

Comment:
Shouldn't all 3 commits be squashed into one?

Reply:
> Shouldn't all 3 commits be squashed into one?

Sorry, I'm not a programmer guy. I squashed commits into one. I hope it was done correctly.